pages/projects/[id]/index.tsx

```diff
@@ -19,7 +22,38 @@ interface Props {
 export const ProjectPage = ({ projects, currentProject }: Props) => {
   return (
     <ProjectLayout projects={projects} currentProject={currentProject}>
-      <h1>ProjectPage for {currentProject.name}</h1>
+      <div className="w-full bg-white bg-opacity-10 px-5 py-6">
+        <div className="flex flex-col md:flex-row md:space-x-1">
+          <HiOutlineCommandLine className="h-8 w-8 shrink-0 text-teal-300" />
+          <div className="mt-2 px-0 md:mt-0 md:px-4">
+            <h3 className="text-sm leading-relaxed text-white">
+              This project does not have a production branch defined
+            </h3>
+            <p className="mt-1 text-sm font-light text-gray-500">
+              Production branches are protected. You cannot update it directly
+              without making a pull request.You can unprotect production
+              branches by going to settings page.
+            </p>
+          </div>
+        </div>
+      </div>
+
+      <div className="mt-8 w-full">
+        <div className="flex w-full items-center justify-between">
+          <Button outline small>
+            main
+          </Button>
```

Comment:
Lets replace this with Popover with searchable select (similar to what we have on project picker, top navigation).